pkg/api/config.go

```diff
@@ -134,3 +139,39 @@ func (c *Config) Validate(log log.Logger) error {
 
 	return nil
 }
+
+// LoadAccessControlConfig populates config.AccessControl struct with values from config.
+func (c *Config) LoadAccessControlConfig() error {
```

Comment:
I don't like this one.
Because I removed the repositories key, I Unmarshall into c.HTTP.RawAccessControl and then manually populate c.AccessControl.

Another approach I used was with a global variable, but that was even uglier.
If only viper supported Custom Unmarshall on specific fields...
If you have a better idea, please let me know.

Thank you.